.github/actions/build-dotnet/action.yml

```diff
@@ -132,10 +141,19 @@ runs:
             Version = "${{ inputs.publish-version }}"
             Switches = $buildSwitches
             ExpectedCodeAnalysisErrors = $expectedCodeAnalysisErrors
+            CreateBinaryLog = $${{ inputs.create-binary-log }}
         }
 
         $startTime = [DateTime]::Now
         Build-DotNetSolutionOrProject @switches
         $endTime = [DateTime]::Now
 
         Write-Output ("Solution or project build took {0:0.###} seconds." -f ($endTime - $startTime).TotalSeconds)
+
+    - name: Upload MSBuild Binary Log
+      uses: actions/upload-artifact@v3.1.1
```

Comment:
Is there a way to disable zipping the binlog? Because it doesn't save any space, and is just cumbersome when downloaded. (I'm exaggerating a bit, of course, but it would still be nice to simply download the file with the name instead of a zip archive.)

Reply:
No, having a zip is mandatory: https://github.com/actions/upload-artifact#zipped-artifact-downloads

Reply:
Thank you. Simple API, good API.